File: lessons_v2/lesson02/codes/nq027_acw672.cpp
#include <cstdio>
int main() {
    double x;
    scanf("%lf", &x);
    if (x <= 2000) { printf("Isento\n"); return 0; }
    x -= 2000;
    double t = 0;
    if (x > 0) { double v = x < 1000 ? x : 1000; t += v * 0.08; x -= v; }
    if (x > 0) { double v = x < 1500 ? x : 1500; t += v * 0.18; x -= v; }
    if (x > 0) t += x * 0.28;
    printf("R$ %.2lf\n", t);
    return 0;
}

File: lessons_v2/lesson02/codes/nq027_acw672.py
# NQ: AcWing 672
x = float(input())
if x <= 2000:
    print('Isento')
else:
    x -= 2000
    t = 0
    if x > 0:
        v = min(x, 1000)
        t += v * 0.08
        x -= v
    if x > 0:
        v = min(x, 1500)
        t += v * 0.18
        x -= v
    if x > 0:
        t += x * 0.28
    print(f"R$ {t:.2f}")
